### ISSUE TITLE
Search finds wrong item when playing on non-default player

### ISSUE THREAD

**Issue by mattpurnell** [maykar/plex_assistant#144]
Hi, the plex_assistant is great when using the default player. Excellent addition.

However it is very hit and miss when trying to play to other media players. An example is in the logs below: I request "boiler room" and it plays a seemingly unrelated documentary "Absolute Zero". Considering the wording in the request, I find that a bit surprising :)

It happens over and over with this film, even if the request text is "play the movie boiler room on the Bedroom Cast.

There are multiple players in the bedroom (android bravias have several players recognised by plex) but I get the same result regardless of how I refer to them.

**Install**: installed integration in Home Assistant docker
**Trigger**: mqtt calling the Home Assistant "plex_assistant.command", or the HA command directly (same results)
**Logs**:

```2021-09-12 13:03:29 DEBUG (SyncWorker_1) [custom_components.plex_assistant.const] Command: play boiler room on the Bedroom TV
2021-09-12 13:03:29 DEBUG (SyncWorker_1) [custom_components.plex_assistant.const] Processed Command: {'media': 'boiler room', 'device': 'bedroom tv'}
2021-09-12 13:03:29 DEBUG (SyncWorker_1) [custom_components.plex_assistant.const] PA Devices: {'Bravia Cast': {'entity_id': 'media_player.plex_bravia', 'device_type': 'plex'}, 'Bedroom speaker': {'entity_id': 'media_player.bedroom_speaker', 'device_type': 'cast'}, 'Bedroom Cast': {'entity_id': 'media_player.sonycast', 'device_type': 'cast'}, 'Lounge': {'entity_id': 'media_player.lounge', 'device_type': 'cast'}, 'Lounge speaker': {'entity_id': 'media_player.lounge_speaker', 'device_type': 'cast'}, 'Bedroom Plex': {'entity_id': 'media_player.plex_bedroomcast', 'device_type': 'plex'}, 'Plex (Plex for Android (Mobile) - SM-G975F)': {'entity_id': 'media_player.plex_plex_for_android_mobile_sm_g975f', 'device_type': 'plex'}}
2021-09-12 13:03:29 DEBUG (SyncWorker_1) [custom_components.plex_assistant.const] Device: Bedroom Cast
2021-09-12 13:03:29 DEBUG (SyncWorker_1) [custom_components.plex_assistant.const] Media: [<Episode:36431:Absolute-Zero-s01e01>, <Episode:36432:Absolute-Zero-s01e02>]```

**Comment by maykar** [comment]
This is a strange one as it's understanding and parsing the correct title `'media': 'boiler room'`. Going to require some digging.

**Comment by mattpurnell** [comment]
Hi - thanks for looking into it.

Yesterday I removed and re-added the cast integration from Home Assistant. In the process i had to remove and readd the Plex assistant.

I also changed the friendly-name for the casts, which means any subsequent queries on my part have been different.

Since then - touching wood - it has been accurate! So, strange as it was, it seems to have resolved, or maybe been avoided. If I can no longer reproduce it, I don't see the requirement in keeping this issue open?

Cheers

And PS - very glad I've found this integration, it's awesome
